white/think-about-where-the-journey-is-going.md

```diff
@@ -0,0 +1,19 @@
+# Think about where the journey is going
+
+Depending on the criticality of your own software, achieving the highest Security Belt might not make economic sense and is therefore not necessary.
+
+- Estimate which Security Belt might be sufficient for the business criticality of your software.
+- Commit on your estimation with your product owner.
+- Challenge your estimation after each achieved Security Belt. Since you should have a better understanding of security, your estimation should be more profound.
```

Comment:
```suggestion
- Challenge your estimation after each achieved Security Belt. Since you should have a better understanding of security, your estimation should become more profound.
```